zedd-app/src/AppState.ts

```diff
@@ -245,8 +245,13 @@ export class AppState {
     ),
   )
   public slices: IObservableArray<TimeSlice> = observable([])
+
   public markedSlices: IObservableArray<TimeSlice> = observable([])
 
+  public sliceToCopy: TimeSlice | undefined
+  @observable
+  public copiedSlice: TimeSlice | undefined = undefined
```

Comment:
Warum hast du zwei Variablen, die ähnliches speichern. Worin besteht der Unterschied in den Variablen? Vielleicht eindeutiger benennen.